Makefile.work

```diff
@@ -109,9 +109,9 @@ SONIC_BUILD_INSTRUCTION :=  make \
                            BUILD_TIMESTAMP=$(BUILD_TIMESTAMP) \
                            ENABLE_DHCP_GRAPH_SERVICE=$(ENABLE_DHCP_GRAPH_SERVICE) \
                            SHUTDOWN_BGP_ON_START=$(SHUTDOWN_BGP_ON_START) \
-                           SONIC_ENABLE_PFCWD_ON_START=$(ENABLE_PFCWD_ON_START) \
-                           SONIC_ENABLE_SYNCD_RPC=$(ENABLE_SYNCD_RPC) \
-                           SONIC_INSTALL_DEBUG_TOOLS=$(INSTALL_DEBUG_TOOLS) \
+                           ENABLE_PFCWD_ON_START=$(ENABLE_PFCWD_ON_START) \
+                           ENABLE_SYNCD_RPC=$(ENABLE_SYNCD_RPC) \
+                           INSTALL_DEBUG_TOOLS=$(INSTALL_DEBUG_TOOLS) \
```

Comment:
this is not correct. if ENABLE_PFCWD_ON_START is not defined, then it will override the default setting in rules/config.